core/citrus-base/src/main/java/org/citrusframework/actions/ReceiveMessageAction.java

```diff
@@ -418,7 +418,7 @@ public MessageBuilder getMessageBuilder() {
     /**
      * Action builder.
      */
-    public static final class Builder extends ReceiveMessageActionBuilder<ReceiveMessageAction, ReceiveMessageActionBuilderSupport, Builder> {
+    public static class Builder extends ReceiveMessageActionBuilder<ReceiveMessageAction, ReceiveMessageActionBuilderSupport, Builder> {
```

Comment:
I don't see why the removal of `final` is required by this commit but I guess it is a preparation for upcoming changes. Please double check if this is required in upcoming PRs